roles/arteria-sequencing-report-ws/templates/pipeline_configs/demultiplex.yml.j2

```diff
@@ -0,0 +1,26 @@
+---
+
+main_workflow_path: "{{ sw_path }}/demultiplex/workflow/main.nf"
+# Note that in the following sections it is possible to do variable subsitution
+# on the following variables:
+# - {runfolder_name}
+# - {runfolder_path}
+# - {current_year}
+# - {input_samplesheet}
+nextflow_parameters:
+  config: "{{ arteria_service_nextflow_config_dir }}/demultiplex.config"
+  profile: "singularity,uppmax"
+environment:
+  NXF_TEMP: {{nextflow_env.NXF_TEMP}}
+  NXF_WORK: "{runfolder_path}/.nextflow"
+  NXF_SINGULARITY_CACHEDIR: {{ ngi_containers }}/demultiplex
```

Comment:
I would say that it's preferred to put the singularity containers in `biocontainers`, i.e. (I believe) `{{ ngi_containers }}/biocontainers`, unless you have a specific reason to keep them separate. It's an older pattern to have specific subdirectories (or for specific non-container dependencies)